circle.yml

```diff
@@ -2,1364 +2,314 @@ version: 2.1
-          command: make ios-check-events-symbols
-      - run:
-          name: Nitpick Darwin code generation
-          command: scripts/nitpick/generated-code.js darwin
-      - save-dependencies
-      - collect-xcode-build-logs
-      - upload-xcode-build-logs
-
-# ------------------------------------------------------------------------------
-  ios-debug-xcode11:
-    macos:
-      xcode: "11.0.0"
-    environment:
-      BUILDTYPE: Debug
-      HOMEBREW_NO_AUTO_UPDATE: 1
-      HOMEBREW_NO_INSTALL_CLEANUP: 1
-    steps:
-      - install-macos-dependencies
-      - install-dependencies
-      - check-public-symbols
-      - run:
-          name: Lint podspecs and plist files
-          command: make ios-lint
-      - check-if-this-job-can-be-skipped
-      - build-ios-test
-      - run:
-          name: Check symbol namespacing for mapbox-events-ios
-          command: make ios-check-events-symbols
-      - run:
-          name: Nitpick Darwin code generation
-          command: scripts/nitpick/generated-code.js darwin
-      - save-dependencies
-      - collect-xcode-build-logs
-      - upload-xcode-build-logs
-
-# ------------------------------------------------------------------------------
-  ios-sanitize-nightly:
-    macos:
-      xcode: "10.3.0"
-    environment:
-      BUILDTYPE: Debug
-      HOMEBREW_NO_AUTO_UPDATE: 1
-      HOMEBREW_NO_INSTALL_CLEANUP: 1
-      SLACK_CHANNEL: C0ACM9Q2C
-    steps:
-      - install-macos-dependencies
-      - install-dependencies
-      - run:
-          name: Build and run SDK unit tests with thread and undefined behavior sanitizers
-          command: make ios-sanitize
-      - run:
-          name: Get iOS code coverage
-          command: |
-              platform/ios/scripts/ios-code-coverage.sh CI
-      - save-dependencies
-      - collect-xcode-build-logs
-      - upload-xcode-build-logs
-      - notify-slack-nightly-failure
-
-# ------------------------------------------------------------------------------
-  ios-sanitize-address-nightly:
-    macos:
-      xcode: "10.3.0"
-    environment:
-      BUILDTYPE: Debug
-      HOMEBREW_NO_AUTO_UPDATE: 1
-      HOMEBREW_NO_INSTALL_CLEANUP: 1
-      SLACK_CHANNEL: C0ACM9Q2C
-    steps:
-      - install-macos-dependencies
-      - install-dependencies
-      - run:
-          name: Build and run SDK unit tests with address sanitizer
-          command: make ios-sanitize-address
-      - save-dependencies
-      - collect-xcode-build-logs
-      - upload-xcode-build-logs
-      - notify-slack-nightly-failure
-
-# ------------------------------------------------------------------------------
-  ios-static-analyzer-nightly:
+      UBSAN_OPTIONS: print_stacktrace=1:halt_on_error=1
+      MSAN_OPTIONS: poison_in_dtor=1
+      ASAN_OPTIONS: strict_string_checks=1:detect_stack_use_after_return=1:check_initialization_order=1:strict_init_order=1
+  macos-11_0_0:
     macos:
-      xcode: "10.3.0"
+      xcode: '11.0.0'
```

Comment:
parameterize this var and pick a generic name so we can reuse this with Xcode 10 and 11.